index.js

```diff
@@ -98,7 +99,7 @@ function send(ctx, path, opts) {
     ctx.set('Last-Modified', stats.mtime.toUTCString());
     ctx.set('Content-Length', stats.size);
     ctx.set('Cache-Control', 'max-age=' + (maxage / 1000 | 0));
-    ctx.type = type(path);
+    ctx.type = typeOverride ? typeOverride : type(path);
     ctx.body = fs.createReadStream(path);
```

Comment:
Overriding the `content type` for the `whole` served directory?
If you want to set `contnt-type` for special file, you can

```js
yield send(this, this.path);
if (...) {
  this.type = 'some'
}
```